doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -23,12 +23,61 @@ Relatedly, C-level executives and public policy professionals often make, shape,
 
 ## Enumerating Vulnerability Management Actions
-In this paper, we model the decision of “With what priority should the organization take action on a given vulnerability management work unit?” to be agnostic to whether or not a patch is available. A unit of work means either remediating the vulnerability, such as, applying a patch or deploying a mitigation. Both remediation  and mitigations often address multiple identified vulnerabilities. The deployer should answer the suggested questions for whatever unit of work they are considering as a whole, single unit. There is not necessarily a reliable function to aggregate a recommendation about remediation out of its constituent vulnerabilities. For the sake of simplicity of examples, we treat the remediation as a patch of one vulnerability, and comment on any difficulty in generalizing our advice to a more complex patch where appropriate. 
+In this paper, we model the decision of “With what priority should the organization take action on a given vulnerability management work unit?” to be agnostic to whether or not a patch is available. 
+A unit of work means either remediating the vulnerability, such as applying a patch or deploying a mitigation. Both remediation and mitigations often address multiple identified vulnerabilities. 
 
-To further clairify terms, "Remediaton occurs when the vulnerability is eliminated or removed. Mitigation occurs when the impact of the vulnerability is decreased without reducing or eliminating the vulnerability." [@dodi_8531_2020, section 3.5] Examples of remediation includes, applying patches, fixes and upgrades; or removing the vulnerable software or system from operation. Mitigating acions may include, software configuration changes, adding firewall ACLs or otherwise limiting the system's exposure to reduce the risk of the impact of the vulnerability; or accepting the risk.
+The unit of work may be different for different stakeholders.
+The units of work can also change as the vulnerability response progresses through a stakeholder's process.
+We elucidate these ideas with the following examples.
 
-### Supplying Patches
+### Supplier Units of Work
+
+On the input side of the Supplier process, Suppliers typically receive reports of vulnerabilities in one or more versions of their product. 
+Part of the Supplier's task on initial report intake is to resolve the initial report into a set of products and versions that are affected by the reported vulnerability. 
+On the output side, Suppliers provide fix bundles for affected products.
```

Comment:
I think this output can reflect both remediation and mitigation. 

> ... Suppliers provide either remediations or mitigations. 
A supplier-provided remediation is usually a software update which contains fixes for multiple vulnerabilities and, often, new or improved features. 
Supplier output is relevant because it will become Deployers' input. 
SSVC focuses just on the remediation in this case; a set of remediations for multiple vulnerabilities is a **fix bundle**. 
Suppliers may also produce mitigations, such as recommended configuration changes, to limit the impact of a vulnerability.

Reply:
fixed in a520c0917272dc76675d7d21a2e250b17edd4de1